### PR TITLE
API integration tests

### PR DESCRIPTION
**Pull request by Vavilen**
- /address_uxouts
- /block
- /last_blocks
- /network/connections
- /network/connection
- /network/defaultConnections
- /network/connections/trust
- /network/connections/exchange

### CODE REVIEW

**Comment by Vavilen** [comment]
#948

File: src/api/cli/integration/integration_test.go [hunk 1]
```diff
@@ -41,7 +42,6 @@ var (
 )
 
 var (
-	update     = flag.Bool("update", false, "update golden files")
```
Comment (Vavilen) [inline src/api/cli/integration/integration_test.go]: Got varcheck error:
src/api/cli/integration/integration_test.go:45:2:warning: unused variable or constant update (varcheck)

Comment (therealssj) [reply src/api/cli/integration/integration_test.go]: Can you undo this change? It was defined but not used yet https://github.com/skycoin/skycoin/pull/993, using it in this PR

Comment (Vavilen) [reply src/api/cli/integration/integration_test.go]: @therealssj , OK. I`ll return it and wait until #993 will be merged.

**Comment by therealssj** [comment]
`src/gui/integration/test-fixtures/blockchain-180.db` This file is changed everytime the tests are run but those changes don't need to be pushed.

**Comment by therealssj** [comment]
@Vavilen src/gui/integration/test-fixtures/blockchain-180.db` This file is changed everytime the tests are run but those changes don't need to be pushed.

**Comment by gz-c** [comment]
merge conflicts

File: src/gui/integration/integration_test.go [hunk 2]
```diff
@@ -601,6 +613,251 @@ func TestLiveUxOut(t *testing.T) {
...
+		addr         string
+		moreThanZero bool
+	}{
+		{
+			name:    "no addresses",
+			errCode: http.StatusBadRequest,
+			errMsg:  "400 Bad Request - address is empty\n",
+		},
+		{
+			name:    "no addresses",
```
Comment (iketheadore) [inline src/gui/integration/integration_test.go]: name: invalid address lenght

File: src/gui/integration/integration_test.go [hunk 3]
```diff
@@ -601,6 +613,251 @@ func TestLiveUxOut(t *testing.T) {
...
+}
+
+func TestBlocks(t *testing.T) {
+	if !doLiveOrStable(t) {
+		return
+	}
+	c := gui.NewClient(nodeAddress())
+	progress, err := c.BlockchainProgress()
+	require.NoError(t, err)
+	blocks, err := c.Blocks(1, int(progress.Current))
```
Comment (iketheadore) [inline src/gui/integration/integration_test.go]: This will allocate lots of memory in live test.

**Comment by gz-c** [comment]
merge conflict

**Comment by gz-c** [comment]
#1022

**Pull request closed without merging**